dealer.rb

```diff
@@ -0,0 +1,13 @@
+$LOAD_PATH << '.'
+require 'player'
+
+# Class for dealer, a type of player with extra rules - must hit until 17
+class Dealer < Player
+  def move?
+    if total <= 17
+      true
+    else
+      false
+    end
```

Comment:
Since you are returning true/false based off the result of the if statement, what do you think about just returning the result of the comparison?

```
    def move?
       total <= 17
    end
```